deploy/bin/gen-ed25519-key.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const tweetnacl = require('tweetnacl');
+
+function main() {
+  console.log('Generating ed25519 keypair...');
+  let keypair = tweetnacl.sign.keyPair();
+
+  fs.writeFile('docker-worker-ed25519.pub', new Buffer.from(keypair.publicKey).toString('base64'), function(err) {});
+  return fs.writeFile('docker-worker-ed25519-cot-signing-key.key', new Buffer.from(keypair.secretKey).toString('base64'), function(err) {});
```

Comment:
Maybe it is better to use [writeFileSync](https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options) here.